functional_tests/features/release_page.feature

```diff
@@ -9,31 +9,18 @@ Feature: CMS users can draft, edit, and publish release pages
         When the user clicks "Add child page" to create a new draft release page
         Then the date placeholder, "YYYY-MM-DD HH:MM", is displayed in the date input textboxes
 
-    Scenario: A CMS user can author and publish release page
-        When the user clicks "Add child page" to create a new draft release page
-        And the user sets the page status to "Published"
```

Comment:
As this test involved setting the page status to "Published" I have removed it, as this is not allowed anymore